config/settings.php

```diff
@@ -1,112 +1,111 @@
 <?php
 
 return [
+
+    'defaults' => [
+        'general' => [
+            'name' => env('APP_NAME', 'PILOS'),
```

Comment:
check possibility of changing to DEFAULT_APP_NAME

Reply:
Not possible, added note to docs to also change .env file